t/ui/PhantomTest.pm

```diff
@@ -106,9 +106,13 @@ sub call_phantom() {
     return $_driver = start_phantomjs($mojoport);
 }
 
-sub wait_for_ajax() {
+sub wait_for_ajax {
+    my ($check_interval) = (@_);
+    if (!$check_interval) {
```

Comment:
Since you've already merged it, I'll leave it as it is. Next time I'm going to use it.